ui/components-react/src/test/editors/TextEditor.test.tsx

```diff
@@ -68,7 +68,11 @@ describe("<TextEditor />", () => {
         propertyDescription: PropertyDescription,
         value?: Primitives.Value | undefined
       ): string | Promise<string> {
-        return super.convertPropertyToString(propertyDescription, `-${value}`);
+        return super.convertPropertyToString(
+          propertyDescription,
+          // eslint-disable-next-line @typescript-eslint/no-base-to-string
```

Comment:
I guess this ignore is not necessary after change to `string(value)`?

Reply:
Linter still complains, due to potential `[object Object]` when stringified.